lms/views/error.py

```diff
@@ -26,9 +26,8 @@ def __init__(self, exc, request):
     def httperror(self):
         self.request.response.status_int = self.exc.status_int
         # If code raises an HTTPError or HTTPServerError we assume this was
-        # deliberately raised and:
-        # 1. Show the user an error page including specific error message
-        # 2. _Do not_ report the error to Sentry
+        # deliberately raised and show the user an error page including specific error message
+        self.request.raven.captureException()
```

Comment:
Can you provide some explanation for why we're making this change? The comment that was removed says that if code explicitly raises `HTTPError` or `HTTPServerError`, this implies that it was intentional, therefore not a bug and therefore not reported to Sentry. This makes sense.

Presumably there is some context in which the current behaviour is not desirable. What is that?